teamup-server/src/main/java/com/project/teamup/utils/MailService.java

```diff
@@ -0,0 +1,135 @@
+package com.project.teamup.utils;
+
+import com.project.teamup.dto.UserDTO;
+import com.project.teamup.model.User;
+
+import javax.mail.*;
+import javax.mail.internet.InternetAddress;
+import javax.mail.internet.MimeMessage;
+import java.util.Date;
+import java.util.Properties;
+
+/**
+ * Sends mails to different users of the application.
+ * @author Sonya
+ * @since 23.11.2019
+ * */
+public class MailService {
+
+    /**
+     * Sends an email
+     *
+     * @param userEmail email of the user who must receive the message
+     * @param subject the subject of the mail
+     * @param message the email's message
+     * @throws MessagingException if the mail hasn't been sent
+     * */
+    public static void sendMail(String userEmail, String subject, String message) throws MessagingException {
+        String username = "teamUp732@gmail.com";
```

Comment:
Please add this configuration as a global configuration for the mail settings. You can put them in application.properties

Reply:
Please add these in enviroment.ts (as @popiosifcristian said)